graphql_api/tests/hazard/setup_helpers.py

```diff
@@ -365,12 +365,12 @@ def create_scaled_solution(self, upstream_sid, task_id):
         print(result)
         return result
 
-    def create_aggregate_solution(self, upstream_sids, task_id, aggregation_fn):
+    def create_aggregate_solution(self, upstream_sids, task_id, aggregation_fn, common_rupture_set):
         """test helper"""
         query = '''
-            mutation ($source_solutions: [ID!], $produced_by: ID!, $digest: String!,
+            mutation ($source_solutions: [ID]!, $produced_by: ID!, $digest: String!,
```

Comment:
@chrisdicaprio note the subtle change in the query - due to making the field `required=True`